plugins/woocommerce/src/Internal/Admin/Agentic/AgenticWebhookPayloadBuilder.php

```
@@ -0,0 +1,230 @@
+	private function get_additional_metadata( $order ) {
+		return array(
+			'order_id'       => $order->get_id(),
+			'order_key'      => $order->get_order_key(),
+			'currency'       => $order->get_currency(),
+			'total'          => $this->format_amount( $order->get_total() ),
+			'customer_email' => $order->get_billing_email(),
+			'created_via'    => $order->get_created_via(),
+			'date_created'   => $order->get_date_created() ? $order->get_date_created()->format( 'c' ) : null,
+			'date_modified'  => $order->get_date_modified() ? $order->get_date_modified()->format( 'c' ) : null,
+		);
+	}
```

PHPMD (2.15.0)

Avoid unused private methods such as 'get_additional_metadata'. (undefined)

(UnusedPrivateMethod)